web/src/main/react-dashboard/src/components/DeploymentPatternView.js

```diff
@@ -63,33 +62,78 @@ class DeploymentPatternView extends Component {
     this.props.history.push(route);
   }
 
+
   render() {
     console.log(this.state);
+    var x = {};
```

Comment:
Shall we name the variable properly? 

Console.log should not be in production - https://stapp.space/how-to-remove-javascript-console-log-pros-2/